src/platform/plugins/private/presentation_panel/public/panel_component/panel_header/use_hover_actions_styles.tsx

```diff
@@ -0,0 +1,157 @@
+        transition: none; // apply transition delay on hover out only
+      }
+    `;
+  }, [euiTheme, showBorder, isEditMode]);
+
+  const hoverActionStyles = useMemo(() => {
+    const singleWrapperStyles = css`
+      width: fit-content;
+      top: -${euiTheme.size.l} !important;
+      right: ${euiTheme.size.xs};
+      padding: var(--paddingAroundAction);
+
+      border-radius: ${euiTheme.border.radius.medium};
+      border: var(--internalBorderStyle);
+      background-color: ${euiTheme.colors.backgroundBasePlain};
+      grid-template-columns: max-content;
+
+      & > * {
+        // undo certain styles on all children so that parent takes precedence
+        border: none !important;
+        padding: 0px !important;
+        border-radius: unset !important;
+        background-color: transparent !important;
+        height: unset !important;
+      }
+    `;
+
+    return css`
+      --paddingAroundAction: calc(${euiTheme.size.xs} - 1px);
+
+      pointer-events: none; // prevent hover actions wrapper from blocking interactions with other panels
+
+      height: ${euiTheme.size.xl};
+      padding: 0px ${euiTheme.size.m};
+      width: 100%;
+
+      display: flex;
+      align-items: center;
+
+      & > * {
+        height: ${euiTheme.size.xl};
+
+        &:not(#embPanel__hoverActionsBreakpoint) {
+          flex: 0; // do not grow
+          pointer-events: all; // re-enable pointer events for non-breakpoint children
+          // style children that are **not** the breakpoint
+          border-top: var(--internalBorderStyle);
+          border-radius: 0px;
+          background-color: ${euiTheme.colors.backgroundBasePlain};
+          padding: var(--paddingAroundAction) 0px;
+        }
+      }
+
+      & > #embPanel__hoverActionsBreakpoint {
+        flex: 1; // grow to fill remaining space between left and right action groups
+      }
+
+      // start of action group
+      & > *:first-child:not(#embPanel__hoverActionsBreakpoint),
+      & > #embPanel__hoverActionsBreakpoint + * {
+        border-left: var(--internalBorderStyle);
+        border-top-left-radius: ${euiTheme.border.radius.medium} !important;
+        padding-left: var(--paddingAroundAction);
+      }
+
+      // end of action group
+      & > *:has(+ #embPanel__hoverActionsBreakpoint),
+      & > *:last-child {
+        border-right: var(--internalBorderStyle);
+        border-top-right-radius: ${euiTheme.border.radius.medium} !important;
+        padding-right: var(--paddingAroundAction);
+      }
+
+      // shrink down to single wrapped element with no breakpoint when panel gets small
+      @container hoverActionsAnchor (width < 250px) {
+        ${singleWrapperStyles}
+      }
+
+      // when in fullscreen mode, combine all floating actions on first row and nudge them down
+      .dshDashboardViewportWrapper--isFullscreen .dshDashboardGrid__item[data-grid-row='0'] & {
+        ${singleWrapperStyles}
+        top: -${euiTheme.size.s} !important;
+      }
```

Comment:
I was trying to avoid having **any** styles within hover actions that referenced dashboard-related styling; however, I couldn't find a workaround for this one in particular without super overcomplicating things 😢